js/components/ReportSummary.vue

```diff
@@ -62,7 +62,7 @@ import ReportSummaryForm from './ReportSummaryForm';
 
 export const messages = {
   missingReport: 'The report used for this summary no longer exists.',
-  missingBucketByField: 'The field used to group counts by no longer exists, has been renamed, or was removed from the report.'
+  missingBucketByField: 'The field used to group counts by no longer exists, has been renamed, or was removed from the report ', // append report title to this message
```

Comment:
It would have been nice if these could have been template strings, but didn't seem worth the extra effort at this time.